src/app/api/public/builder/events/page/route.ts

```diff
@@ -53,24 +46,7 @@ interface PageEventBody {
   previousValue?: EventValue
 }
 
-export const POST = withRouteMiddleware(async (request: NextRequest) => {
-  const authHeader = request.headers.get('Authorization')
-
-  if (authHeader !== `Bearer ${BUILDER_IO_WEBHOOK_AUTH_TOKEN}`) {
-    Sentry.captureMessage('Received unauthorized request to Builder.io webhook', {
-      extra: {
-        ...request,
-      },
-      tags: {
-        domain: 'builder.io',
-      },
-    })
-
-    return new NextResponse('Unauthorized', {
-      status: 401,
-    })
-  }
-
+export const POST = withBuilderIoAuthMiddleware(async (request: NextRequest) => {
```

Comment:
You shouldn't remove `withRouteMiddleware` from this route, IIRC it adds some important sentry metadata to the request. You can nest these higher order function calls no problem (or you should be able to if the typings are set correctly)
```suggestion
export const POST = withRouteMiddleware(withBuilderIoAuthMiddleware(async (request: NextRequest) => {
```

Reply:
does it even make sense to have it around a webhook called by builder.io?